temporal/fx.go

```diff
@@ -268,6 +271,16 @@ func ServerOptionsProvider(opts []ServerOption) (serverOptionsProvider, error) {
 		}
 	}
 
+	// static service hosts
+	staticHosts := make(map[primitives.ServiceName][]string)
+	if so.staticHosts {
+		for serviceName := range so.serviceNames {
+			staticHosts[serviceName] = []string{
+				fmt.Sprintf("127.0.0.1:%v", so.config.Services[string(serviceName)].RPC.GRPCPort),
```

Comment:
To avoid making this localhost-specific, can we have the server option accept the whole map? This could potentially be useful in other situations.

Reply:
Sure 👍